.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -12,6 +12,15 @@ model_proto = onnx.load("path to .onnx file")
 checker.check_model(model_proto)
 ```
 
+### Contribute a Gradio Demo to ONNX Organization on Hugging Face
+
+* Create an account on Hugging Face: https://huggingface.co/join
+* See list of models left to add to ONNX organization, please refer to the table with the [Models list](https://github.com/onnx/models)
```

Comment:
```suggestion
* See list of models left to add to ONNX organization, please refer to the table with the [Models list](https://github.com/onnx/models#models)
```
nit: link to more precise section. L22 ditto.

Reply:
@jcwchen thanks updated